components/Investment/TrancheMetric/index.tsx

```diff
@@ -1,48 +1,63 @@
 import * as React from 'react';
 import { Box, Heading } from 'grommet';
-import { baseToDisplay, Investor, Tranche } from 'tinlake';
-import NumberDisplay from '../../NumberDisplay';
+import { Investor, Tranche } from 'tinlake';
 import { TrancheType } from '../../../services/tinlake/actions';
 import DashboardMetric from '../../DashboardMetric';
+import { Erc20Widget } from '../../../components/erc20-widget';
+import DAI from '../../../static/dai.json';
 
 interface Props {
   investor: Investor;
   tranche: Tranche;
   type: TrancheType;
+  tinlake: any;
 }
 
 class TrancheMetric extends React.Component<Props> {
+
   render() {
     const { type, investor, tranche } = this.props;
     const { maxSupply, maxRedeem, tokenBalance } = investor[type];
     const { token  } = tranche;
-    const currencyLabel = ` ${token}`;
-
+    const dropAddress = this.props.tinlake.contractAddresses.SENIOR_TOKEN as string;
+    const tinAddress = this.props.tinlake.contractAddresses.JUNIOR_TOKEN as string;
+    const dropToken = {
+      [dropAddress] : {
+        symbol: 'DROP',
+        logo: '../../static/DROP_final.svg',
+        decimals: 18,
+        name: 'DROP'
+      }
+    };
+    const tinToken = {
+      [tinAddress] : {
+        symbol: 'TIN',
+        logo: '../../static/TIN_final.svg',
+        decimals: 18,
+        name: 'TIN'
+      }
+    };
     return <Box margin="none">
       <Box>
         <Heading level="4" margin={{ bottom: 'medium' }}>Investment overview</Heading>
         <Box direction="row" >
-          <Box basis={'1/3'} gap="medium">
             <DashboardMetric label="Investor token balance">
-              <NumberDisplay value={tokenBalance ? baseToDisplay(tokenBalance, 18) : ''} suffix={currencyLabel} precision={18} />
+              {token === 'DROP' && <Erc20Widget value={tokenBalance ? tokenBalance.toString() : '0'} tokenData={dropToken} precision={18}  />}
```

Comment:
If you make the distinction in the investorView container based you won't need to do it here